src/components/railBottom/controls/SubmenuPopper/SubmenuPopper.vue

```diff
@@ -1,45 +1,94 @@
 <template>
   <popper
-    trigger="clickToToggle"
+    v-if="items"
+    trigger="clickToOpen"
     :options="{
       placement: 'top',
       modifiers: { offset: { offset: '0,20px' } }
     }"
     :visible-arrow=false
+    :force-show="popperType === data.type"
   >
-    <div v-if="data.items">
+    <div>
       <ul class="control-submenu">
-        <li v-for="(item, key) in data.items" class="control-submenu-list"  :key="key">
-          <img :src=item.iconSrc :alt=item.label>
-          <div>
+        <li v-for="(item, key) in items"
+          :class="{'control-submenu-list active': selectedItem === item.type, 'control-submenu-list': selectedItem !== item.type}"
```

Comment:
same comment goes for the simplified version

Reply:
It will be refactored